### ISSUE TITLE
Is it possible to do a release/tag?

### ISSUE THREAD

**Issue by rmannibucau** [dcasati/kubernetes-PlantUML#9]
Hi,

would be neat to have stable links in diagrams and therefore a tag which can be imported (define) instead of relying on master.

**Comment by turesheim** [comment]
Yes, using https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.2.0/C4_Container.puml I now get an error:
`!unquoted procedure $defineSkinparams($tagStereo, $bgColor, $fontColor, $borderColor, $shadowing)
  !$tagSkin = $elementTagSkinparams("rectangle", $tagStereo, $bgColor, $fontColor, $borderColor, $shadowing)
^^^^^
 Unknown variable $tagStereo`
Similar when trying any other released version of C4-PlantUML, just a different error.
